definitions/tools/sequence_to_fastq_rna.cwl

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Picard: BAM to FASTQ"
+baseCommand: ["/bin/bash","makefastqs.sh"]
+requirements:
+    - class: ResourceRequirement
+      coresMin: 1
+      ramMin: 16000
+      tmpdirMin: 25000
+    - class: DockerRequirement
+      dockerPull: "mgibio/rnaseq:1.0.0"
```

Comment:
(Obviously this isn't something changed for this PR, but in the long run it might be nice if we switched this to a picard image instead.)